.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - v8.8
+    - v8.9
+    - v8.10
+    - v8.11
+    - v8.12
+    - v8.13
+  pull_request:
+    branches:
+    - v8.8
+    - v8.9
+    - v8.10
+    - v8.11
+    - v8.12
+    - v8.13
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        ocaml-version: [4.07.1, 4.08.1, 4.09.1, 4.10.1, 4.11.1]
+        test-target: [test]
+        extra-opam: ["coq.8.13.0 coq-mathcomp-ssreflect"]
+        include:
+        - ocaml-version: 4.07.1+32bit
+          test-target: js-dune
+          extra-opam: coq.8.13.0 js_of_ocaml js_of_ocaml-lwt zarith_stubs_js
+        - ocaml-version: 4.11.1
+          test-target: test
+          extra-opam: dune
```

Comment:
Shouldn't this be captured by the `opam install --deps-only` below?

Reply:
Idk, I was just copying the Travis script

Reply:
Ok, I'll review once the checks actually run.